### ISSUE TITLE
Behavior of get method for non-active relationships

### ISSUE THREAD

**Issue by yoavnash** [simphony/simphony-osp#424]
Currently, the default behavior of the `get` only returns objects that are linked via an active relationship. This could lead to a surprising/unintuitive behavior, especially for non-containment relationships.

Example:
``` python
alice = my_onto.person(name="Alice")
bob = my_onto.person(name="Bob")
alice.add(bob, rel=my_onto.friends_with) # friends_with is a non-containment (and symmetrical) relationship

# more code ...

# get Alice's friends:
bob_retrieved = alice.get(bob.uid)
# bob_retrieved is equal to None, although Alice is still friends with Bob.
```

#### Proposed solution 1
Extend the behavior of the `get` method so that it also returns object linked with non-containment relationships, while still ignoring those that are linked in a passive relationship. In other words, the default relationship of `get` should not be active_relationship, but rather the complement of passive_relationship.

#### Proposed solution 2
Extend the behavior of the `get` method so that it returns all connected objects. This is more intuitive in the sense that here `get()` is more general than `get(rel=onto.is_part_of)` as one might expect (it's not the case with the current behavior). Possible downside: this change will perhaps encourage less thinking in terms of containers, which is a central idea in OSP-core.

**Comment by urbanmatthias** [comment]
I agree that solution two is the simplest. However it feels like we are - step by step - loosing the container characteristic of the CUDS objects.
